pinc/wordcheck_engine.inc

```diff
@@ -60,16 +60,16 @@ define('WC_PAGE', 4);
  * @param string $projectid
  *   ID of project, needed for project languages and to load the custom
  *   dictionaries
- * @param string $aux_languages
- *   Auxiliary language to check against.
+ * @param array $languages
+ *   Languages to check against.
  * @param array $adhoc_good_words
  *   Array of words to treat as good for this invocation only.
  *
  * @return array
  */
-function get_bad_word_freqs_for_project_text($text, $projectid, $aux_language = '', $adhoc_good_words = [])
+function get_bad_word_freqs_for_project_text($text, $projectid, $languages, $adhoc_good_words = [])
```

Comment:
Removing a default was intentional since the underlying function no longer uses `$projectid` to generate a set, so I wanted to force callers to explicitly either include a list or set it to `[]` (which would be odd given the function, but legal).